docs/modules/ROOT/pages/how-to-guides/proc_maintaining_operator_graphs.adoc

```diff
@@ -0,0 +1,222 @@
+│
+├─ Directory for the targetted OCP version (for example, 4.12)
+│   ├─ devfile.yaml
+│   └─ catalog
+        └─ catalog.yaml
+|
+├─ Directory for the targetted OCP version (for example, 4.13)
+│   ├─ devfile.yaml
+│   └─ catalog
+        └─ catalog.yaml
+│
+└─ README.md
+----
+
+
+* *`catalog.Dockerfile`*: This file contains the instructions for building the FBC image. It should specify a parent image, such as `registry.redhat.io/openshift4/ose-operator-registry`. The Dockerfile should also have a tag corresponding to the target OCP version. 
+
++	
+*Example:*
+
++
+[source,catalog.Dockerfile]
+----
+# The base image is expected to contain
+# /bin/opm (with a serve subcommand) and /bin/grpc_health_probe
+FROM registry.redhat.io/openshift4/ose-operator-registry:v4.12
+
+# Configure the entrypoint and command
+ENTRYPOINT ["/bin/opm"]
+CMD ["serve", "/configs", "--cache-dir=/tmp/cache"]
+
+# Copy declarative config root into image at /configs and pre-populate serve cache
+ADD catalog /configs
+RUN ["/bin/opm", "serve", "/configs", "--cache-dir=/tmp/cache", "--cache-only"]
+
+# Set DC-specific label for the location of the DC root directory
+# in the image
+LABEL operators.operatorframework.io.index.configs.v1=/configs
+----
+
+* *`catalog.yaml`*: This file contains important metadata about the operator. There should be a `catalog.yml` file for each FBC image that is produced.
+
++
+*Example:*
+
++
+[source,yaml]
+----
+schema: olm.composite.catalogs
+catalogs:
+  - name: "4.13" <1>
+    destination:
+      baseImage: registry.redhat.io/openshift4/ose-operator-registry:v4.13 <2>
+      workingDir: .
+    builders:
+      - olm.builder.basic
+  - name: "4.12"
+    destination:
+      baseImage: registry.redhat.io/openshift4/ose-operator-registry:v4.12 This template can be hosted
+      workingDir: .
+    builders:
+      - olm.builder.basic
+----
+<1> The OCP version.
+<2> The parent image that the system uses to build the FBC fragments.
+
+
+* *`devfile.yaml`*: This file is used by {ProductName} to detect the type of component and to set up a build.
+
++	
+*Example:*
+
++
+[source,yaml]
+----
+schemaVersion: 2.2.0
+metadata:
+  name: fbc-4.12
+  displayName: FBC 4.12
+  description: 'File based catalog'
+  language: fbc <1>
+  provider: Red Hat
+components:
+  - name: image-build
+    image:
+      imageName: ""
+      dockerfile:
+        uri: catalog.Dockerfile <2>
+        buildContext: "4.12" <3>
+  - name: kubernetes
+    kubernetes:
+      inlined: placeholder
+    attributes:
+      deployment/container-port: 50051
+      deployment/cpuRequest: "100m"
+      deployment/memoryRequest: 512Mi
+      deployment/replicas: 1
+      deployment/storageRequest: "0"
+commands:
+  - id: build-image
+    apply:
+      component: image-build"
+----
+<1> Indicates the component type that {ProductName} builds.
+<2> Indicates the Dockerfile as defined earlier.
+<3> Indicates the tag corresponding to the targetted OCP version as defined earlier.
```

Comment:
<3> does not indicate the tag -- the tag corresponding to the OCP version is represented in the Dockerfile (i.e. the FROM statement). This is the context directory needed to specify the FBC component corresponding to the OCP version being targeted (i.e. according to the referenced Dockerfile).